deploy-board/manage.py

```diff
@@ -39,20 +39,36 @@
 
     #
     # OAuth based authentication settings. By default, OAuth based authentication is disabled
-    # See documentation for how to enable OAuth
     #
     os.environ.setdefault("OAUTH_ENABLED", "OFF")
+
+    # Uncomment the following to turn on oauth authentication. Make sure to enable authentication,
+    # and possibly authorization on the backend java Teletraan service as well. See Teletraan doc for more details.
+    # See also https://developers.google.com/identity/protocols/OpenIDConnect for more open id authentication
     #os.environ.setdefault("OAUTH_ENABLED", "ON")
-    #os.environ.setdefault("OAUTH_CLIENT_ID", <OAUTH_CLIENT_ID>)
-    #os.environ.setdefault("OAUTH_CALLBACK", "<DEPLOYBOARD_URL>/auth")
-    #os.environ.setdefault("OAUTH_DOMAIN", <YOUR_DOMAIN>)
-    #os.environ.setdefault("OAUTH_CLIENT_TYPE", <CLIENT_TYPE>)
-    #os.environ.setdefault("OAUTH_USER_INFO_URI", <USER_INFO_URL>)
-    #os.environ.setdefault("OAUTH_ACCESS_TOKEN_URL", <ACCESS_TOKEN_URL>)
-    #os.environ.setdefault("OAUTH_AUTHORIZE_URL", <OAUTH_AUTH_URL>)
-    #os.environ.setdefault("OAUTH_DEFAULT_SCOPE", <DEFAULT_SCOP>)
-    # Key for user response, which key is associated with the username in the response. Emails are parsed for prefix.
-    #os.environ.setdefault("OAUTH_USERNAME_INFO_KEY", <USERNAME_KEY>)
+
+    # The client ID that you obtain from the provider
+    #os.environ.setdefault("OAUTH_CLIENT_ID", 'REPLACEME.apps.googleusercontent.com')
```

Comment:
It may be more convenient to define a dict like Google_OAUTH_Settings grouped all Google related settings there and set it. Later each OAuth provider have a similar grouped configs